templates/default/tests/pages.spec.js

```diff
@@ -24,13 +24,13 @@ test.describe("Pages", () => {
   // })
 
   // Test for nested index.md routes
-  test('Nested index routes [test/index.md]', async ({ page, baseURL }) => {
+  test('Nested index routes [demo/index.md]', async ({ page, baseURL }) => {
     const Page = new MarkdownPage(page);
-    await Page.goto("/test");
```

Comment:
I would have kept the test page as we may use it for other things. IMO demos should be for demoing (ie. something we'll show users) and test pages for testing. That said, for now we could merge if that is useful as often doing the same thing. But in that case i'd stick to test i think.

Minor point!